telegram/_bot.py

```diff
@@ -147,7 +147,7 @@ class Bot(TelegramObject, AsyncContextManager["Bot"]):
     Examples:
         :any:`Raw API Bot <examples.rawapibot>`
 
-    .. seealso:: :wiki:`Your First Bot <Extensions-–-Your-first-Bot>`,
+    .. seealso:: :wiki:`Your First Bot <Extensions---Your-first-Bot>`,
```

Comment:
Does that link still work?

Reply:
no :( Maybe let's just change the wiki heading itself then?

Reply:
I think it is better to do this just before release, as if we do it now users will have to click again if they visit the wiki from the docs

Reply:
forgot that we had that..

Reply:
done in live-wiki.. by mistake, little damage anyway